src/KafkaFlow.Retry.IntegrationTests/Core/Storages/InMemoryAuxiliarStorage.cs

```diff
@@ -25,7 +26,7 @@ public static async Task AssertCountMessageAsync(T message, int count)
 
             while (Message.Count(x => x.Key == message.Key && x.Value == message.Value) != count)
             {
-                if (DateTime.Now.Subtract(start).TotalSeconds > TimeoutSec)
+                if (DateTime.Now.Subtract(start).TotalSeconds > TimeoutSec && !Debugger.IsAttached)
```

Comment:
There are others timeout. Does it `&& !Debugger.IsAttached` make sense on them as well.

Reply:
added. Please check the last commit.